### ISSUE TITLE
slickROutput(height = '50px') not respected

### ISSUE THREAD

**Issue by DavorJ** [yonicd/slickR#47]
Since the last version, the `height` parameter of `slickROutput` doesn't seem to work.

Here is an example: 

![image](https://user-images.githubusercontent.com/4161918/91980541-b7b7b980-ed27-11ea-8d24-f0df57fafe17.png)

The blue bounding box is `50px` but as you see, the pictures are extended beyond it.

**Comment by yonicd** [comment]
Thanks for catching this. I'll look into it.

**Comment by yonicd** [comment]
You are correct. the way the underlying js code is written the widget size supersedes the container size it is in. to change this will take some refactoring. For the time being unfortunately you will have to control the size via the slickR function and not  the output function.

**Comment by ismirsehregal** [comment]
It seems that on the other hand the `width` argument is only taken into account when used in `slickROutput` not in `slickR`.
